Create ui-c8y-1023-22-2-updated-ca-certificate-renewal-threshold-from-18-to-24-months-10751.md (#4112)
SoftwareAG/c8y-docs: content/change-logs/platform-services/ui-c8y-1023-22-2-updated-ca-certificate-renewal-threshold-from-18-to-24-months-10751.md

---
date: ""

> title: Updated CA certificate renewal threshold from 18 to 24 months (#10751) [GRAFT][release/cd] (#10809)

Reviewer: Which option sounds best, @BeateRixen?

```suggestion
title: Updated CA certificate renewal threshold from 18 to 24 months
```

- can we use `{{< company-c8y >}}` in the `title`?
```suggestion
title: Updated renewal threshold from 18 to 24 months for CA certificates provided by {{< company-c8y >}}
```
```suggestion
title: Earlier renewal option of CA certificates provided by {{< company-c8y >}}
```

> title: Updated CA certificate renewal threshold from 18 to 24 months (#10751) [GRAFT][release/cd] (#10809)

Reviewer 2: No, we can´t use shortcodes in the title. I think "CA certificate renewal threshold updated to 24 months" is sufficient as the details are explained in the text itself.

> title: Updated CA certificate renewal threshold from 18 to 24 months (#10751) [GRAFT][release/cd] (#10809)

Reviewer: ```suggestion
title: CA certificate renewal threshold updated to 24 months
```

product_area: Platform services
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-0UgqXH1Ys
    label: Administration
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-65709
version: 1023.22.2
---

> As found out during the review of Cumulocity-IoT/c8y-docs#4076, backend changed the mentioned threshold to 24 months, so UI needs to align + linting fixes.

Reviewer: ```suggestion
The renewal option of CA certificates provided by {{< company-c8y >}} is now available up to 2 years before the expiration date. See more details in the [documentation](/docs/device-certificate-authentication/managing-trusted-certificates/#to-renew-a-ca-certificate).
```

> The renewal option of CA certificates provided by {{< company-c8y >}} is now available up to 2 years before the expiration date. See more details in the [documentation](/docs/device-certificate-authentication/managing-trusted-certificates/#to-renew-a-ca-certificate).

Reviewer 2: ```suggestion
The renewal option for CA certificates provided by {{< product-c8y-iot >}} has been extended, allowing renewals up to 2 years before the expiration date, giving you more flexibility in certificate management. For more details, see the [documentation](/docs/device-certificate-authentication/managing-trusted-certificates/#to-renew-a-ca-certificate).
```
